<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IDs3</class>
 <widget class="QWidget" name="IDs3">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>710</width>
    <height>570</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>710</width>
    <height>570</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Gen 3 TID\SID</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_5">
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <widget class="QWidget" name="gen3XDColo">
      <attribute name="title">
       <string>XD/Colo</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBoxXDColoSearchBy">
         <property name="title">
          <string>Search By</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <widget class="QCheckBox" name="checkBoxXDColoPID">
            <property name="text">
             <string>PID</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="TextBox" name="textBoxXDColoPID">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="maxLength">
             <number>8</number>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QCheckBox" name="checkBoxXDColoTID">
            <property name="text">
             <string>TID</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="TextBox" name="textBoxXDColoTID">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="maxLength">
             <number>5</number>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QCheckBox" name="checkBoxXDColoSID">
            <property name="text">
             <string>SID</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="TextBox" name="textBoxXDColoSID">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="maxLength">
             <number>5</number>
            </property>
           </widget>
          </item>
          <item row="3" column="0" colspan="2">
           <widget class="QPushButton" name="pushButtonXDColoSearch">
            <property name="text">
             <string>Search</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QGroupBox" name="groupBoxXDColoParameters">
         <property name="title">
          <string>Search Parameters</string>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="labelXDColoSeed">
            <property name="text">
             <string>Seed</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="TextBox" name="textBoxXDColoSeed">
            <property name="maxLength">
             <number>8</number>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="labelXDColoStartingAdvance">
            <property name="text">
             <string>Initial Advances</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="TextBox" name="textBoxXDColoStartingAdvance">
            <property name="text">
             <string notr="true">5000</string>
            </property>
            <property name="maxLength">
             <number>10</number>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="labelXDColoMaxAdvances">
            <property name="text">
             <string>Max Advances</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="TextBox" name="textBoxXDColoMaxAdvances">
            <property name="text">
             <string notr="true">10000</string>
            </property>
            <property name="maxLength">
             <number>10</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="TableView" name="tableViewXDColo">
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="gen3FRLGE">
      <attribute name="title">
       <string>Gen III FRLGE</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_7">
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBoxFRLGESearchBy">
         <property name="title">
          <string>Search By</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_6">
          <item row="0" column="0">
           <widget class="QCheckBox" name="checkBoxFRLGEPID">
            <property name="text">
             <string>PID</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="TextBox" name="textBoxFRLGEPID">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="maxLength">
             <number>8</number>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QCheckBox" name="checkBoxFRLGESID">
            <property name="text">
             <string>SID</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="TextBox" name="textBoxFRLGESID">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="maxLength">
             <number>5</number>
            </property>
           </widget>
          </item>
          <item row="2" column="0" colspan="2">
           <widget class="QPushButton" name="pushButtonFRLGESearch">
            <property name="text">
             <string>Search</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QGroupBox" name="groupBoxFRGLESearchParameters">
         <property name="title">
          <string>Search Parameters</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="0" column="0">
           <widget class="QLabel" name="labelFRLGETID">
            <property name="text">
             <string>TID</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="TextBox" name="textBoxFRLGETID">
            <property name="maxLength">
             <number>5</number>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="labelFRLGEStartingAdvance">
            <property name="text">
             <string>Initial Advances</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="TextBox" name="textBoxFRLGEStartingAdvance">
            <property name="text">
             <string notr="true">5000</string>
            </property>
            <property name="maxLength">
             <number>10</number>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="labelFRLGEMaxAdvances">
            <property name="text">
             <string>Max Advances</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="TextBox" name="textBoxFRLGEMaxAdvances">
            <property name="text">
             <string notr="true">10000</string>
            </property>
            <property name="maxLength">
             <number>10</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="TableView" name="tableViewFRLGE">
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="gen3RS">
      <attribute name="title">
       <string>Gen III RS</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_10">
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBoxRSSearchBy">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>200</height>
          </size>
         </property>
         <property name="title">
          <string>Search By</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_8">
          <item row="0" column="0">
           <widget class="QCheckBox" name="checkBoxRSPID">
            <property name="text">
             <string>PID</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="TextBox" name="textBoxRSPID">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="maxLength">
             <number>8</number>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QCheckBox" name="checkBoxRSTID">
            <property name="text">
             <string>TID</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="TextBox" name="textBoxRSTID">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="maxLength">
             <number>5</number>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QCheckBox" name="checkBoxRSSID">
            <property name="text">
             <string>SID</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="TextBox" name="textBoxRSSID">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="maxLength">
             <number>5</number>
            </property>
           </widget>
          </item>
          <item row="3" column="0" colspan="2">
           <widget class="QPushButton" name="pushButtonRSSearch">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>23</height>
             </size>
            </property>
            <property name="text">
             <string>Search</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QGroupBox" name="groupBoxRSSearchParameters">
         <property name="title">
          <string>Search Parameters</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_9">
          <item row="1" column="3">
           <widget class="QRadioButton" name="radioButtonRSInitialSeed">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Initial Seed</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0" colspan="2">
           <widget class="QCheckBox" name="checkBoxRSDeadBattery">
            <property name="text">
             <string>Dead Battery</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
            <property name="tristate">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QRadioButton" name="radioButtonRSDate">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Date</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="1" colspan="2">
           <widget class="QDateTimeEdit" name="dateTimeEdit">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="dateTime">
             <datetime>
              <hour>0</hour>
              <minute>0</minute>
              <second>0</second>
              <year>2000</year>
              <month>1</month>
              <day>1</day>
             </datetime>
            </property>
            <property name="maximumDateTime">
             <datetime>
              <hour>23</hour>
              <minute>59</minute>
              <second>59</second>
              <year>2037</year>
              <month>12</month>
              <day>31</day>
             </datetime>
            </property>
            <property name="minimumDateTime">
             <datetime>
              <hour>0</hour>
              <minute>0</minute>
              <second>0</second>
              <year>2000</year>
              <month>1</month>
              <day>1</day>
             </datetime>
            </property>
            <property name="minimumDate">
             <date>
              <year>2000</year>
              <month>1</month>
              <day>1</day>
             </date>
            </property>
            <property name="calendarPopup">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="4">
           <widget class="TextBox" name="textBoxRSInitialSeed">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="maxLength">
             <number>4</number>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="TextBox" name="textBoxRSStartingAdvance">
            <property name="text">
             <string notr="true">5000</string>
            </property>
            <property name="maxLength">
             <number>10</number>
            </property>
           </widget>
          </item>
          <item row="3" column="2">
           <widget class="TextBox" name="textBoxRSMaxAdvances">
            <property name="text">
             <string notr="true">10000</string>
            </property>
            <property name="maxLength">
             <number>10</number>
            </property>
           </widget>
          </item>
          <item row="2" column="0" colspan="2">
           <widget class="QLabel" name="labelRSStartingAdvance">
            <property name="text">
             <string>Initial Advances</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0" colspan="2">
           <widget class="QLabel" name="labelRSMaxAdvance">
            <property name="text">
             <string>Max Advances</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="TableView" name="tableViewRS">
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TextBox</class>
   <extends>QLineEdit</extends>
   <header>Forms/Controls/TextBox.hpp</header>
  </customwidget>
  <customwidget>
   <class>TableView</class>
   <extends>QTableView</extends>
   <header>Forms/Controls/TableView.hpp</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>checkBoxXDColoPID</tabstop>
  <tabstop>textBoxXDColoPID</tabstop>
  <tabstop>checkBoxXDColoTID</tabstop>
  <tabstop>textBoxXDColoTID</tabstop>
  <tabstop>checkBoxXDColoSID</tabstop>
  <tabstop>textBoxXDColoSID</tabstop>
  <tabstop>pushButtonXDColoSearch</tabstop>
  <tabstop>textBoxXDColoSeed</tabstop>
  <tabstop>textBoxXDColoStartingAdvance</tabstop>
  <tabstop>textBoxXDColoMaxAdvances</tabstop>
  <tabstop>tableViewXDColo</tabstop>
  <tabstop>checkBoxFRLGEPID</tabstop>
  <tabstop>textBoxFRLGEPID</tabstop>
  <tabstop>checkBoxFRLGESID</tabstop>
  <tabstop>textBoxFRLGESID</tabstop>
  <tabstop>pushButtonFRLGESearch</tabstop>
  <tabstop>textBoxFRLGETID</tabstop>
  <tabstop>textBoxFRLGEStartingAdvance</tabstop>
  <tabstop>textBoxFRLGEMaxAdvances</tabstop>
  <tabstop>tableViewFRLGE</tabstop>
  <tabstop>checkBoxRSPID</tabstop>
  <tabstop>textBoxRSPID</tabstop>
  <tabstop>checkBoxRSTID</tabstop>
  <tabstop>textBoxRSTID</tabstop>
  <tabstop>checkBoxRSSID</tabstop>
  <tabstop>textBoxRSSID</tabstop>
  <tabstop>pushButtonRSSearch</tabstop>
  <tabstop>checkBoxRSDeadBattery</tabstop>
  <tabstop>radioButtonRSDate</tabstop>
  <tabstop>dateTimeEdit</tabstop>
  <tabstop>radioButtonRSInitialSeed</tabstop>
  <tabstop>textBoxRSInitialSeed</tabstop>
  <tabstop>textBoxRSStartingAdvance</tabstop>
  <tabstop>textBoxRSMaxAdvances</tabstop>
  <tabstop>tableViewRS</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>checkBoxXDColoPID</sender>
   <signal>clicked(bool)</signal>
   <receiver>textBoxXDColoPID</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>37</x>
     <y>76</y>
    </hint>
    <hint type="destinationlabel">
     <x>225</x>
     <y>72</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxXDColoTID</sender>
   <signal>clicked(bool)</signal>
   <receiver>textBoxXDColoTID</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>48</x>
     <y>103</y>
    </hint>
    <hint type="destinationlabel">
     <x>303</x>
     <y>108</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxXDColoSID</sender>
   <signal>clicked(bool)</signal>
   <receiver>textBoxXDColoSID</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>73</x>
     <y>139</y>
    </hint>
    <hint type="destinationlabel">
     <x>222</x>
     <y>137</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxFRLGEPID</sender>
   <signal>clicked(bool)</signal>
   <receiver>textBoxFRLGEPID</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>44</x>
     <y>79</y>
    </hint>
    <hint type="destinationlabel">
     <x>242</x>
     <y>81</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxFRLGESID</sender>
   <signal>clicked(bool)</signal>
   <receiver>textBoxFRLGESID</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>60</x>
     <y>119</y>
    </hint>
    <hint type="destinationlabel">
     <x>241</x>
     <y>121</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxRSPID</sender>
   <signal>clicked(bool)</signal>
   <receiver>textBoxRSPID</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>54</x>
     <y>79</y>
    </hint>
    <hint type="destinationlabel">
     <x>125</x>
     <y>79</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxRSTID</sender>
   <signal>clicked(bool)</signal>
   <receiver>textBoxRSTID</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>73</x>
     <y>120</y>
    </hint>
    <hint type="destinationlabel">
     <x>127</x>
     <y>117</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxRSSID</sender>
   <signal>clicked(bool)</signal>
   <receiver>textBoxRSSID</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>54</x>
     <y>159</y>
    </hint>
    <hint type="destinationlabel">
     <x>150</x>
     <y>162</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
